courses/models.py

```diff
@@ -75,14 +80,28 @@ def __str__(self):
     def get_absolute_url(self):
         return self.quiz.get_absolute_url()
 
-class Answer(models.Model):
-    # Each Question should have exactly 4 answers
+class Choice(models.Model):
+    # Each Question should have exactly 4 choices
     question = models.ForeignKey(Question, on_delete=models.CASCADE)
-    answer_text = models.TextField()
-    is_correct = models.BooleanField(default=False)
+    choice_text = models.TextField()
+    #is_correct = models.BooleanField(default=False)
 
     def __str__(self):
-        return self.answer_text
+        return self.choice_text
+
+class Answer(models.Model):
```

Comment:
Why do we need a new model for Answer?

Reply:
To record the id of the choice that serves as the correct answer of a question.

If I have only two models: Question and Choice, I can't record the id of the choice as the correct answer since choice cannot be created before the question as we don't have the foreignkey (the id of question).